### PR TITLE
Kill version in releases table.

### PR DESCRIPTION
**Pull request by bhearsum**
I also improved the UI in a couple of places:
1) Use "Product: foo" in the main product display, rather than just showing the bolded product.
2) Title the "View Data" modal with release name rather than product name.

### CODE REVIEW

File: auslib/test/admin/views/test_releases.py [hunk 1]
```diff
@@ -735,39 +691,34 @@ def testPostRevisionRollback(self):
...
         count, = query.execute().first()
-        self.assertEqual(count, 2 * 2)
+        self.assertEqual(count, 2)
 
-        # Oh no! We prefer the version 222.0
         row, = table.history.select(
-            where=[table.history.version == '222.0'],
+            where=[table.history.product == 'd'],
             limit=1
         )
```
Comment (nthomas-mozilla) [inline auslib/test/admin/views/test_releases.py]: I think we ought to add some sort of reverse sorting here, to ensure we revert the most-recent change.

File: auslib/test/admin/views/test_releases.py [hunk 2]
```diff
@@ -779,11 +730,10 @@ def testPostRevisionRollback(self):
 
         query = table.history.t.count()
         count, = query.execute().first()
-        self.assertEqual(count, 2 * 2 + 1)
+        self.assertEqual(count, 3)
 
         row, = table.select(where=[table.name == 'd'])
-        self.assertEqual(row['version'], '222.0')
-        self.assertEqual(row['data_version'], 6)
+        self.assertEqual(row['data_version'], 4)
```
Comment (nthomas-mozilla) [inline auslib/test/admin/views/test_releases.py]: Lets add a check that the release blob has been reverted properly. Ideally we can tell the difference between test setup --> True --> False --> <reverted value>, so might need to swap to something numerical over boolean.

**Comment by nthomas-mozilla** [comment]
r+, but please make testPostRevisionRollback a little more robust prior to merging. And just to confirm - a submission script that still sends a version will be fine, that'll be ignored ?

**Comment by bhearsum** [comment]
I'll definitely fix up that test first. That's a very good point about the submission script though...I think it may cause a 400. I'll look into that as well.

**Comment by bhearsum** [comment]
Looks like passing random unknown POST arguments doesn't break anything, so we're good on that front: http://people.mozilla.org/~bhearsum/sattap/2a89a05f.png. I'll be updating the submission script anyways though, of course. I'll push a fix for testPostRevisionRollback in a bit.

**Comment by bhearsum** [comment]
How's this @nthomas-mozilla?

**Comment by nthomas-mozilla** [comment]
Looks good to me.

**Pull request merged**